robottelo/ssh.py

```diff
@@ -315,9 +315,14 @@ def execute_command(cmd, connection, output_format=None, timeout=None,
         else:
             logger.error('ssh command did not respond in the predefined time'
                          ' (timeout=%s) and will be interrupted', timeout)
+            stdout.channel.close()
+            stderr.channel.close()
```

Comment:
what about putting logger here instead of multiline exception?
```
logger.error('Captured stdout]\n{0}\n-----\n'.format(stdout.read()))
logger.error('Captured stderr]\n{0}\n-----\n'.format(stderr.read()))
```

Reply:
i kind of concluded that it would be better to see the output in the errorDetails rather than in the stdout section of the junitxml.
On the second thought, i think you're right and logically it should go there.

Reply:
Lets see what 3rd person inclines to :-)